diesel/src/macros/macros_from_codegen.rs

```diff
@@ -15,7 +19,15 @@ macro_rules! infer_schema {
 /// This macro can only be used in combination with the `diesel_codegen` or
 /// `diesel_codegen_syntex` crates. It will not work on its own.
 ///
-/// FIXME: Oh look we have a place to actually document this now.
+/// Establishes a database connection at compile time, loads the schema information about a table's
+/// columns, and invokes [`table!`](macro.table!.html) for you automatically. Attempting to use the
```

Comment:
It's the URL currently on the website. I'd expect that they will make old URLs continue to work.